std_msgs/String problem
---
trajectory_msgs/JointTrajectory plan